### DISCUSSION TITLE
Custom notifications

### DISCUSSION THREAD

**Discussion by andrewstech** [upptime/upptime#191] [Ideas] [answered]
I very much enjoy this project. I was wondering if it was possible to implement custom notification services that use a webhook.  This would be useful for third party services and the addition of using  triggering software via webhooks.

**Comment by ririko5834** [comment]
Good idea

**Comment by wolfier74** [comment]
I love this idea, add that you can specify request method, request url, request headers and request body in repository secrets

**Comment by coolusaHD** [comment, marked as answer]
This should add the option for simple webhooks.
If this gets merged I can look into add features like @wolfier74 requested

https://github.com/upptime/uptime-monitor/pull/254
